SR2021/2020-11-13-sr2021-kickstart-details.md

```diff
@@ -0,0 +1,38 @@
+---
+to: Potential team leaders + signed up teams
+subject: Student Robotics Kickstart Information
+---
+
+Hello there!
+
+We're pleased to announce some more details for the Student Robotics 2020 kickstart.
+
+As you know, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment this year.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020 at 13:00 GMT**, aiming to finish around 16:00. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+The simulator will be based in [webots](https://cyberbotics.com/#download). The download is large, and we recommend downloading it before the event to ensure you can get started immediately. This is the same simulator as used for the [SR2020 virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), but you should download the latest version. Information on downloading our simulation will be made available on [the docs](https://studentrobotics.org/docs/simulator/) during the event.
```

Comment:
What does the "SR2020 virtual competition" link add?

Reply:
Context? Can't hurt to link out to things. Gives people more of a sense of what a virtual competition looks like

Reply:
It's already been linked above and link styling makes that phrase stand out. If it weren't a link, would you want that phrase to stand out in that paragraph? Does it help someone merely scanning the paragraph find the information they want?